typify-impl/tests/test_github.rs

```diff
@@ -60,6 +59,7 @@ fn test_other() {
           }
     };
     let schema = serde_json::from_value(raw_schema).unwrap();
+    // Note impls argument is ignored
```

Comment:
really?

Reply:
Well, I tried lots of changes to this value, and it had no impact on the test.